i18nconfig.json

```diff
@@ -27,5 +27,6 @@
     ],
     "rtl": [
         "he-IL"
-    ]
+    ],
+    "googleTranslateApiKey": "AIzaSyCxgbMfGPKSJKkW67VUkjl_LNESdXF9xkQ"
```

Comment:
What is that?
The plugins never asked me to use any apiKey.
Is it something new?

Reply:
There's special function in this plugin that can automatically translated strings by Google Translate API.
API Key of Google Translate is required.

Reply:
Cool!